<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogsSettings</class>
 <widget class="QWidget" name="LogsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>486</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="kcfg_save_logs_into_file">
             <property name="text">
              <string>Save logs into file</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="kcfg_logs_filename"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>in path</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="kcfg_logs_filepath"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="kcfg_show_debug">
           <property name="text">
            <string>show debug</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="kcfg_hide_info_logs">
           <property name="text">
            <string>hide information logs</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logs_to_file_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>logs_filename_lineedit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logs_to_file_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>logs_path_lineedit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
